service/src/main/kotlin/no/nav/su/se/bakover/service/AccessCheckProxy.kt

```diff
@@ -579,17 +581,12 @@ open class AccessCheckProxy(
                     return services.revurdering.sendTilAttestering(request)
                 }
 
-                override fun lagBrevutkast(
+                override fun hentBrevutkast(
                     revurderingId: UUID,
-                    fritekst: String,
+                    fritekst: String?,
```

Comment:
Burde vi ikke fjerne fritekst dersom vi kun henter et brevutkast? Hvordan skiller vi på om det er brevutkastet for forhåndsvarslingen eller avsluttingen?

Reply:
Man vil gjerne se brevet med friteksten man har skrevet? dem begge har egne endepunkter for sine brevutkast.